### ISSUE TITLE
No pose supervision?

### ISSUE THREAD

**Issue by AbyssGaze** [feixue94/imp-release#7]
After looking at the code, it seems that I didn't see any estimation or supervision of pose during the training process. Additionally, the "Iterative" mentioned in the paper seems to correspond to the number of iterations for self-cross attention. I'm not sure if I misunderstood this.

**Comment by feixue94** [comment]
The code of pose supervision is very similar to that used in OANet and will be added later. In each iteration, we perform descriptor augmentation (self/cross) and pose estimation, so it is more like how many steps needed to augment the descriptors so as to find good matches. More iterations are used for tough cases and fewer for easy ones.

**Comment by AbyssGaze** [comment]
Thank you for your reply. I have tried to add OANet directly to the matching for pose estimation before, but the performance slightly decreased. I'm looking forward to your pose estimation code. Thank you very much.

**Comment by July0928** [comment]
Hi, i try the pose supervision on the superglue by learnable coefficient, but the result is not good. I want to confirm if the loss function formed between the generated R、t and groundtruth is through L2Loss? Would you release the code for constructing the loss function section, if possible?
